event/event.go

```diff
@@ -1,96 +1,104 @@
-/* STRUCTURES OF REAL LIFE EVENTS */
+// event defines the structures for real-life events
 package event
 
 import "hash"
 
-//TODO refaire les comments de cette page
-
-/*
-Data Types:
-For readability (human) and proper encoding for JSON, these data types must be transmitted
-and/or displayed in the given format but are stored in bytes.
-*/
-
+// DEPRECATED : subscribers are not stored in a database anymore
 const SubscribeDB = "sub.db"
 
```

Comment:
Add a comment on top explaining hashes/concatenation? i.e
// All concatenation for hashes mentionned in this document are based on the following concept 
`["item1", "item2"]`